spec/features/event_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.feature "Draft events", :type => :feature do
```

Comment:
During merging into `dev`, you may wanna move this either into its own file or as a `describe` within the enclosing `Event` block.